bbb-common-web/src/main/java/org/bigbluebutton/api/model/entity/Metadata.java

```diff
@@ -4,7 +4,7 @@
 import java.util.Objects;
 
 @Entity
-@Table(name = "metadata")
+@Table(name = "src/metadata")

```

Comment:
This should still be just "metadata" it is referring to the name of the table in the database, not a directory.